src/Native/Runtime/StackFrameIterator.cpp

```diff
@@ -79,7 +80,7 @@ GVAL_IMPL_INIT(PTR_VOID, g_RhpRethrow2Addr, PointerToRhpRethrow2);
 #ifdef DACCESS_COMPILE
 #define EQUALS_RETURN_ADDRESS(x, func_name) ((x) == g_ ## func_name ## Addr)
 #else
-#define EQUALS_RETURN_ADDRESS(x, func_name) (((x)) == (PointerTo ## func_name))
+#define EQUALS_RETURN_ADDRESS(x, func_name) (((PINSTRToPCODE(dac_cast<TADDR>(x)))) == (PINSTRToPCODE(dac_cast<TADDR>(PointerTo ## func_name))))
```

Comment:
It would be better to maintain `m_ControlPC` with the thumbit set instead of adjusting for it here.

Adjustments like these tend to spread everywhere when the code addresses may or may not have the thumb-bit set.

Reply:
I updated `SetControlPC` so that it clears the thumb bit and reverted this change and this one:
https://github.com/dotnet/corert/blob/5f4dd60e6130127e2c5871c2d81723e519097cfe/src/Native/Runtime/unix/UnixNativeCodeManager.cpp#L149